api/src/openapi/schemas/observation.ts

```diff
@@ -10,21 +10,13 @@ export const observationUpdateGetResponseObject = {
 /**
  * Basic response object for an observation.
  */
-export const observationIdResponseObject = {
-  title: 'Observation Response Object',
+export const observationResponseObject = {
+  title: 'observation Object',
   type: 'object',
   required: ['id'],
   properties: {
-    blocks: {
-      type: 'array',
-      items: {
-        type: 'object',
-        properties: {
-          id: {
-            type: 'number'
-          }
-        }
-      }
+    id: {
+      type: 'number'
```

Comment:
this should not change, I dont understand why it has been modified
the endpoint is returning an object with blocks property not just an id